chart/templates/cleanup/cleanup-cronjob.yaml

```diff
@@ -38,9 +38,9 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
-{{- with .Values.labels }}
-{{ toYaml . | indent 4 }}
-{{- end }}
+  {{- if or (.Values.labels) (.Values.cleanup.labels) }}
+  {{- mustMerge .Values.cleanup.labels .Values.labels | toYaml | nindent 4 }}
+  {{- end }}
```

Comment:
@jedcunningham I tested this by adding labels to both global labels and to container specific labels. It is showing both in the helm template.
An alternative to merge would be using two if conditions

Reply:
I think it makes sense to include both labels global level and component specific.
This would give more flexibility to the user. All generic labels can be added at global level and specific ones can be added to each component.